src/argilla/client/feedback/schemas/records.py

```diff
@@ -194,9 +195,7 @@ class FeedbackRecord(BaseModel):
     fields: Dict[str, Union[str, None]]
     metadata: Dict[str, Any] = Field(default_factory=dict)
     responses: List[ResponseSchema] = Field(default_factory=list)
-    suggestions: Union[Tuple[SuggestionSchema], List[SuggestionSchema]] = Field(
-        default_factory=tuple, allow_mutation=False
-    )
+    suggestions: Union[Tuple[SuggestionSchema], List[SuggestionSchema]] = Field(default_factory=tuple)
```

Comment:
I think I already mentioned this, but the `tuple` may be confusing, I'm more comfortable with the `list`, in any case, `responses` is still a list, so we should align that at some point

Reply:
I agree, but we need to do this taking into account that tuples have been used in current releases. My change here just removes the `allow_mutation=True`. Other extra things should be tackled in separate PRs. Otherwise, a lot of changes could be included here without a need.